### ISSUE TITLE
Installation fails for Existing Server (Advanced) because of dmidecode

### ISSUE THREAD

**Issue by dmitry-kulikov** [jlund/streisand#738]
### Expected behavior:

Installation completed.

### Actual Behavior:

Installation interrupted because of error:
```
TASK [Try to determine localhost Cloud provider name from BIOS version] ******************************************************************************
fatal: [x.x.x.x]: FAILED! => {"changed": false, "cmd": "dmidecode -s bios-version", "failed": true, "msg": "[Errno 2] No such file or directory", "rc": 2}
```

### Steps to Reproduce:

Run `./streisand`, choose `Existing Server (Advanced)`.

### Additional Details:

Fix is `sudo apt-get install dmidecode` on target host. Although Scaleway is not supported, issue can happen for supported providers in future.

**Comment by cpu** [comment]
Hi @dmitry-kulikov - thanks for reporting! I'm excited to see folks using the localhost provisioning support already.

> Fix is sudo apt-get install dmidecode on target host. Although Scaleway is not supported, issue can happen for supported providers in future.

Good catch. The providers I tested all included `dmidecode` in their base 16.04 instance but I'm happy to explicitly require the package be installed to support cases where that isn't true. I'll have a PR out to fix this shortly.

**Comment by cpu** [comment]
Should be fixed with #740. Thanks again!
